frontend/packages/knative-plugin/src/components/overview/__tests__/EventSinkServicesOverviewList.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as utils from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ResourceLink } from '@console/internal/components/utils';
+import { ServiceModel } from '@console/knative-plugin';
+import EventSinkServicesOverviewList from '../EventSinkServicesOverviewList';
+import { mockEventSourceData } from '../__mocks__/overview-knative-mock';
+
+describe('EventSinkServicesOverviewList', () => {
+  it('should component exists', () => {
+    const wrapper = shallow(<EventSinkServicesOverviewList obj={mockEventSourceData} />);
+    expect(wrapper.exists()).toBeTruthy();
+  });
+
+  it('should have title Knative Services', () => {
+    const wrapper = shallow(<EventSinkServicesOverviewList obj={mockEventSourceData} />);
+    expect(wrapper.find(utils.SidebarSectionHeading)).toHaveLength(1);
+    expect(
+      wrapper
+        .find(utils.SidebarSectionHeading)
+        .at(0)
+        .props().text,
+    ).toEqual('Knative Services');
+  });
+
+  it('should show error info if no sink present or sink,kind is incorrect', () => {
+    const wrapper = shallow(<EventSinkServicesOverviewList obj={} />);
+    expect(wrapper.find('span')).toHaveLength(1);
+    expect(
+      wrapper
+        .find('span')
+        .at(0)
+        .props().className,
+    ).toEqual('text-muted');
+    expect(
+      wrapper
+        .find('span')
+        .at(0)
+        .props().children,
+    ).toEqual('No services found for this resource.');
```

Comment:
Can this simply be `expect(wrapper.text().includes('No services found for this resource.')).toBe(true)`
That way we're not bound to the markup itself.
Not sure it has much value to test that the class `text-muted` is set because it's not conditional.